### PR TITLE
DataGrid - AI Assistant: command execution e2e tests

### PR DESCRIPTION
**Pull request by Alyar666**
## What does the PR change?

Adds e2e TestCafe tests for the **DataGrid AI Assistant** — command execution and grid-state edge cases — single/multi commands, local data parity, stacked operations, and empty/single-row/large/single-column/all-hidden/already-in-target-state grids (plan §1.2–§1.4, §3.1–§3.7).

Part of splitting the AI Assistant e2e suite into per-area pull requests.

### CODE REVIEW

File: e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts [hunk 1]
```diff
@@ -0,0 +1,1136 @@
...
+  { id: 3, name: 'Charlie', value: 10 },
+];
+
+const twoRows = [
+  { id: 1, name: 'Alice', value: 30 },
+  { id: 2, name: 'Bob', value: 20 },
+];
+
+const setupAIState = ClientFunction((base: Record<string, unknown>, responses: unknown[]) => {
+  (window as any).__aiBase = base;
```
Comment (dmirgaev) [inline e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts]: This definition is very often repeated, it'd better move it into a separate definition and use everywhere instead

File: e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts [hunk 2]
```diff
@@ -0,0 +1,1136 @@
...
+  return createWidget('dxDataGrid', remoteAIGridOptions);
+};
+
+// === §1.2 Single-command request, successful execution ===
+
+fixture`AI Assistant - Single Command`
+  .page(AI_INTEGRATION_PAGE);
+
+// 1.2.1
+test('Single sorting command should execute successfully and update grid', async (t) => {
```
Comment (dmirgaev) [inline e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts]: The same thing with comments and fixtures: lets make one fixture per single file and remove excessive comments, as fixture and test descriptions should be enough to understand each test case

File: e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts [hunk 3]
```diff
@@ -0,0 +1,1008 @@
...
+  const aiChat = dataGrid.getAIAssistantChat();
+
+  await t
+    .typeText(aiChat.getInput(), 'Clear sorting')
+    .pressKey('enter');
+
+  await t.expect(aiChat.getSuccessMessages().count).eql(1);
+  await t.expect(aiChat.getSuccessActionItems(0).count).eql(1);
+
+  await t.expect(dataGrid.apiColumnOption('name', 'sortOrder')).notOk();
```
Comment (anna-shakhova) [inline e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts]: eql(undefined)

File: e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts [hunk 4]
```diff
@@ -0,0 +1,1008 @@
...
+  await t.expect(aiChat.getSuccessActionItems(0).count).eql(1);
+  await t.expect(dataGrid.apiOption('filterValue')).eql(['value', '>', 20]);
+
+  await t
+    .typeText(aiChat.getInput(), 'Clear filter')
+    .pressKey('enter');
+
+  await t.expect(aiChat.getSuccessMessages().count).eql(2);
+  await t.expect(aiChat.getSuccessActionItems(1).count).eql(1);
+  await t.expect(dataGrid.apiOption('filterValue')).notOk();
```
Comment (anna-shakhova) [inline e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts]: eql(undefined) - notOk allows all falsy values

File: e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts [hunk 5]
```diff
@@ -0,0 +1,1008 @@
...
+  columns: ['id', 'name', 'value'],
+  showBorders: true,
+  paging: { pageSize: 10 },
+}, [
+  { actions: [{ name: 'sorting', args: { dataField: 'name', sortOrder: 'asc' } }] },
+  { actions: [{ name: 'grouping', args: { dataField: 'name', groupIndex: 0 } }] },
+  { actions: [{ name: 'pageIndex', args: { pageIndex: 1 } }] },
+]));
+
+test('Sort on empty grid should succeed with no rows', async (t) => {
```
Comment (anna-shakhova) [inline e2e/testcafe-devextreme/tests/dataGrid/common/aiAssistant/commands.functional.ts]: that is what is meant by empty grid, i'd remove

**Review by anna-shakhova — DISMISSED** [review]

**Review by Raushen — APPROVED** [review]
